docs/cudf/source/basics.rst

```diff
@@ -0,0 +1,40 @@
+| Integer                |            | `int <https://docs.python.org/3/library/functions.html#int>`_                       | ``'int8'``, ``'int16'``, ``'int32'``,       |
+|                        |            |                                                                                     | ``'int64'``, ``'uint8'``, ``'uint16'``,     |
+|                        |            |                                                                                     | ``'uint32'``, ``'uint64'``                  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Float                  |            | `float <https://docs.python.org/3/library/functions.html#float>`_                   | ``'float32'``, ``'float64'``                |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Strings                |            | `str <https://docs.python.org/3/library/stdtypes.html#str>`_                        | ``'string'``, ``'object'``                  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Datetime               |            | `numpy.datetime64 <https://numpy.org/doc/stable/reference/arrays.datetime.html>`_   | ``'datetime64[s]'``, ``'datetime64[ms]'``,  |
+|                        |            |                                                                                     | ``'datetime64[us]'``, ``'datetime64[ns]'``  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Timedelta              |            | `numpy.timedelta64 <https://numpy.org/doc/stable/reference/arrays.datetime.html>`_  | ``'timedelta64[s]'``, ``'timedelta64[ms]'``,|
+| (duration type)        |            |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Categorical            |            | (none)                                                                              | ``'category'``                              |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Boolean                |            | `bool <https://docs.python.org/3/library/functions.html#bool>`_                     | ``'bool'``                                  |
```

Comment:
Similarly do we not use a numpy scalar for this?

Reply:
Updated the content and links